{
 "PRed": [
  {
   "PR": {
    "number": 25,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Noarch",
    "migrator_version": 0
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version"
   ]
  },
  {
   "PR": {
    "number": 27,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "CompilerRebuild",
    "migrator_version": 1,
    "name": "Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 29,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.4.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 31,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1579272527.9616141,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 32,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 33,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 34,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 2,
    "migrator_version": 0,
    "name": "python39"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 35,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy37"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 36,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1614799715.8431947,
    "migrator_name": "ArchRebuild",
    "migrator_version": 1,
    "name": "aarch64 and ppc64le addition"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 37,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "OSXArm",
    "migrator_version": 1,
    "name": "arm osx addition"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 38,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "ArchRebuild",
    "migrator_version": 1,
    "name": "aarch64 and ppc64le addition"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 40,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.5.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 41,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.6.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": {
  "exception": "Could not clone git@github.com:regro-cf-autotick-bot/jupyter_nbextensions_configurator-feedstock.git - does the remote exist?",
  "traceback": [
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 161, in _run_git_command",
   "    return subprocess.run(",
   "           ^^^^^^^^^^^^^^^",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/subprocess.py\", line 571, in run",
   "    raise CalledProcessError(retcode, process.args,",
   "subprocess.CalledProcessError: Command '['git', 'clone', '--quiet', 'git@github.com:regro-cf-autotick-bot/jupyter_nbextensions_configurator-feedstock.git', PosixPath('feedstocks/jupyter_nbextensions_configurator-feedstock')]' returned non-zero exit status 128.",
   "",
   "The above exception was the direct cause of the following exception:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 191, in clone_repo",
   "    self._run_git_command([\"clone\", \"--quiet\", origin_url, target_dir])",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/contextlib.py\", line 81, in inner",
   "    return func(*args, **kwds)",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 165, in _run_git_command",
   "    raise GitCliError(\"Error running git command.\") from e",
   "conda_forge_tick.git_utils.GitCliError: Error running git command.",
   "",
   "The above exception was the direct cause of the following exception:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 320, in clone_fork_and_branch",
   "    self.clone_repo(origin_url, target_dir)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/contextlib.py\", line 81, in inner",
   "    return func(*args, **kwds)",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 193, in clone_repo",
   "    raise GitCliError(",
   "conda_forge_tick.git_utils.GitCliError: Error cloning repository from git@github.com:regro-cf-autotick-bot/jupyter_nbextensions_configurator-feedstock.git. Does the repository exist? Is target_dir empty?",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 566, in _run_migrator_on_feedstock_branch",
   "    migrator_uid, pr_json = run(",
   "                            ^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 196, in run",
   "    feedstock_dir, repo = get_repo(",
   "                          ^^^^^^^^^",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/contextlib.py\", line 81, in inner",
   "    return func(*args, **kwds)",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 744, in get_repo",
   "    backend.clone_fork_and_branch(",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/contextlib.py\", line 81, in inner",
   "    return func(*args, **kwds)",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 459, in clone_fork_and_branch",
   "    self.cli.clone_fork_and_branch(",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/contextlib.py\", line 81, in inner",
   "    return func(*args, **kwds)",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 323, in clone_fork_and_branch",
   "    raise GitCliError(",
   "conda_forge_tick.git_utils.GitCliError: Could not clone git@github.com:regro-cf-autotick-bot/jupyter_nbextensions_configurator-feedstock.git - does the remote exist?",
   ""
  ]
 },
 "pinning_version": "2022.11.15.22.41.08",
 "pre_pr_migrator_attempts": {},
 "pre_pr_migrator_status": {},
 "smithy_version": "3.21.3"
}